src/Microsoft.AspNetCore.Mvc.Razor/Internal/ViewComponentTagHelperDescriptorFactory.cs

```diff
@@ -56,7 +57,6 @@ public IEnumerable<TagHelperDescriptor> CreateDescriptors(string assemblyName)
                     StringComparison.Ordinal));
 
             var tagHelperDescriptors = viewComponentDescriptors
-                .Where(descriptor => !descriptor.Parameters.Any(parameter => parameter.ParameterType.GetTypeInfo().IsGenericType))
```

Comment:
There can't be open generic parameter types because this is about an actual method invocation.